Photon_TallytoVtk.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct  3 23:18:20 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import matplotlib.pyplot as plt
+
+with openmc.StatePoint("statepoint.10.h5") as sp:
+    flux_spectrum = sp.get_tally(id=2) 
+    mesh=sp.meshes[1]
+    # Get the reshaped tally data
+    tally_data_reshaped = flux_spectrum.get_reshaped_data(value='mean')
+
+    flux_sum_en = tally_data_reshaped.sum(axis=(0,1,3,4,5))   
+    #Vitamin-J energy filter:
+    e_filter = flux_spectrum.filters[2]
+    #Lower bounds of the energy bins
+    e_filter_lower = e_filter.bins[:, 0]

```

Comment:
A lot of magic numbers here including tally ID, mesh number, axes for sum, etc...  more readable if these are named as variables, and might make sense if they lived in a function that took some of these as arguments.